frames/src/main/java/com/checkout/frames/component/expirydate/ExpiryDateComponentState.kt

```diff
@@ -6,4 +6,22 @@ internal data class ExpiryDateComponentState(
     val inputState: InputComponentState
 ) {
     val expiryDate = inputState.inputFieldState.text
+
+    fun hideError() = switchErrorVisibility(false)
+
+    fun showError(errorMessageId: Int) = with(inputState.errorState) {
```

Comment:
Please can you add unit test

Reply:
@deepesh-vasthimal-cko  I have divided tasks into smaller PR. So, Unit tests for this component would be separate PR. You can find this ticket for the unit test. [/PIMOB-1404](https://checkout.atlassian.net/browse/PIMOB-1404)